src/main/java/no/nav/familie/inntektsmelding/database/tjenester/ForespørselTjeneste.java

```diff
@@ -0,0 +1,23 @@
+package no.nav.familie.inntektsmelding.database.tjenester;
+
+import java.time.LocalDate;
+import java.util.UUID;
+
+import no.nav.familie.inntektsmelding.koder.Ytelsetype;
+import no.nav.familie.inntektsmelding.typer.AktørId;
+import no.nav.familie.inntektsmelding.typer.FagsakSaksnummer;
+import no.nav.familie.inntektsmelding.typer.Organisasjonsnummer;
+
+
+public interface ForespørselTjeneste {
```

Comment:
Hvorfor trenger vi et interface hvis det bare brukes et sted?

Reply:
Det var for å ha eit tydeligere skilje mellom database og web